oci/spec_opts.go

```diff
@@ -393,12 +396,72 @@ func WithImageConfigArgs(image Image, args []string) SpecOpts {
 			// even if there is no specified user in the image config
 			return WithAdditionalGIDs("root")(ctx, client, c, s)
 		} else if s.Windows != nil {
+			// imageExtended is a subset of the oci Image struct that changes
```

Comment:
Nit: isn't it a superset?

Reply:
LOL. You are right. I stopped 2 classes short of a math major and did CS instead... You can't blame me :)

Reply:
Fixed.